internal/appsec/waf.go

```diff
@@ -105,7 +105,7 @@ func newWAFEventListener(handle *waf.Handle, addresses []string, appsec *appsec)
 				spanCtx := span.Context()
 				event = withSpanContext(event, spanCtx.TraceID(), spanCtx.SpanID())
 				// Keep this span due to the security event
-				span.SetTag(ext.ManualKeep, true)
+				span.SetTag(ext.AppSecKeep, true)
```

Comment:
`ext.AppSecKeep` is a public tag which is not good, we can just use a string key.

Reply:
Hello,

I just stumbled upon this PR and wanted to let yo know that AppSecKeep was dismissed in internal discussions in favor of ManualKeep + a reason tag (not implemented here)

The AppSec system tests are testing this behaviour and enforcing it across all libaries.

So I am a bit surprised by this.

Reply:
Hi @Julio-Guerra , thank you for taking a look. This PR is still not ready, and was created for an initial feedback.
Yes, if we have the reason, then we can use `ext.ManualKeep`.

The point is, we need to know who made the decision. Was it a customer or AppSec?